extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -907,8 +909,7 @@ private static ParquetInstructions infer(
                     "Unable to infer schema for a partitioned parquet table when there are no initial parquet files");
         }
         final Pair<List<ColumnDefinition<?>>, ParquetInstructions> schemaInfo = ParquetSchemaReader.convertSchema(
-                lastKey.getFileReader().getSchema(),
-                lastKey.getMetadata().getFileMetaData().getKeyValueMetaData(),
```

Comment:
This is one of the logic changes; previously, we were getting the schema from the file instead of from the metadata file.